lib/empty-example/sketch.js

```diff
@@ -1,7 +1,12 @@
+let video;
 function setup() {
- // put setup code here
+  createCanvas(640, 480, WEBGL);
+  video = createCapture(VIDEO, { flipped: true });
+  video.hide();
 }
 
 function draw() {
-  // put drawing code here
+  translate(-width / 2, -height / 2);
+  scale(-1, 1); //added scale for horizontal flip
+  image(video, -width, 0); //here used -width that shifts the video back into view from the opposite side
 }
```

Comment:
hmm...You may need to make some adjustments to the actual code. For instance, when you write the same code in sketch.js:
```js
let video;

function setup() {
  createCanvas(640, 480, WEBGL);
  video = createCapture(VIDEO, { flipped: true });
  video.hide();
}

function draw() {
  translate(-width / 2, -height / 2);
  image(video, 0, 0);
}
```
The aim is to keep the code unchanged, but ensure the output is as expected. To achieve this, it might be necessary to modify the codebase. 

I have a strong feeling the `createCapture` function may not be handling the flipping correctly in `WEBGL`, so it’s worth investigating. Please feel free to reach out if I could help you with any other way for this issue! You don't need to update the `sketch.js` but probably the function used in sketch.js is needed to be changed so that with the same code above we could achieve the output.

Here's the `createCapture` function which could be worth looking at:
https://github.com/processing/p5.js/blob/7d393ec4215ce20f7bc3178b779e0ec67689f928/src/dom/dom.js#L2401

Reply:
Yes, If i am not wrong it will be how webgl is applying texture for video we need to handle that for flipped videos.